client/src/components/App.jsx

```diff
@@ -24,11 +26,14 @@ class App extends React.Component {
       reviews: [],
       averages: [],
       loadAll: false,
+      showPhotos: false,
```

Comment:
Wow, very clean and concise state! Nice work, this makes it much easier to read your code.